jjb/dynamic/uperf.yml

```diff
@@ -0,0 +1,102 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+        # Disable logging
+        set +x
+        oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        # Run tests
+        set -x
+        git clone https://github.com/cloud-bulldozer/plow.git
```

Comment:
We can skip this as we are cloning the plow repo using the github jenkins plugin ( recommended way ) in line 96.